packages/esm-appointments-app/src/appointments.test.tsx

```diff
@@ -10,7 +10,7 @@ describe('Appointments', () => {
     expect(screen.getByRole('button', { name: /appointments calendar/i })).toBeInTheDocument();
     expect(screen.getByPlaceholderText(/dd-mmm-yyyy/i)).toBeInTheDocument();
     screen.getByRole('combobox', {
-      name: /select service type/i,
+      name: /view/i,
```

Comment:
This fixes a failing test in appointments.

Reply:
nvm, [this PR](https://github.com/openmrs/openmrs-esm-patient-management/pull/1389#pullrequestreview-2475939889) has a more substantial fix. Will remove.